appcues/src/main/java/com/appcues/debugger/ui/CaptureConfirmDialog.kt

```diff
@@ -225,22 +225,22 @@ private fun CaptureContents(debuggerViewModel: DebuggerViewModel, capture: Captu
 
 @Composable
 private fun TextWebLink(text: String, @Suppress("SameParameterValue") url: String) {
-    val uriHandler = LocalUriHandler.current
-    ClickableText(
```

Comment:
migrating away from `ClickableText` to `Text` with a `LinkAnnotation` within was a deprecation warning https://joebirch.co/android/migrating-from-the-clickabletext-composable-to-linkannotation/